rdr_service/api/nph_participant_biobank_order_api.py

```diff
@@ -0,0 +1,103 @@
+import json
+import logging
+from flask import request
+
+from werkzeug.exceptions import BadRequest, NotFound
+from sqlalchemy import exc
+
+from rdr_service.api.base_api import UpdatableApi
+from rdr_service.dao import database_factory
+from rdr_service.dao.study_nph_dao import NphOrderDao
+
+
+class CustomEncoder(json.JSONEncoder):
+    def default(self, o):
+        return o.__dict__
+
+
+def construct_response(order):
+    # Construct Response payload
+    return json.loads(json.dumps(order, indent=4, cls=CustomEncoder))
+
+
+class NphOrderApi(UpdatableApi):
+
+    def __init__(self):
+        super(NphOrderApi, self).__init__(NphOrderDao())
+
+    def update_with_patch(self, id_, resource, expected_version):
+        pass
+
+    def put(self, nph_participant_id, rdr_order_id):
```

Comment:
I know you're working on some more unit tests, but I noticed something else we'll want to make sure to update. Only Hpro should be able to make requests to these endpoints, so we'll want to use the `@app_util.auth_required(RTI_AND_HEALTHPRO)` decorator. This should be added to `post` and `patch` as well.

Reply:
Adding the auth_required for RTI and HealthPro